go/client/prompts.go

```diff
@@ -42,4 +42,7 @@ const (
 	PromptDescriptorProvisionDeviceName
 	PromptDescriptorExportSecretKeyFromGPG
 	PromptDescriptorDeprovisionWhichUser
+	PromptDescriptorUpdateDo
+	PromptDescriptorUpdateAuto
+	PromptDescriptorUpdateSkipVersion
```

Comment:
What if we marked an update critical on the server, and warn the user?  Security folks will get upset if we force them to upgrade.